template/.drone.yml

```diff
@@ -7,21 +7,39 @@
 
 matrix:
```

Comment:
In my opinion, the goal is to get rid of the different drone `matrix` build definitions (like at https://github.com/allaboutapps/go-starter/blob/master/.drone.yml#L8-L14 there should only be a single definition).

This should be possible if we no longer rely on docker build-args (compile/generation time) but instead inject all config via `ENV` anyways and `sed` exchange them during runtime

Reply:
After discussion we will remove build time arguments and solely rely on runtime args.
Thanks for the awesome feedback @majodev!